tests/health_checks/NDv4/check-hardware.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
Just a suggestion to move these files currently under tests/health_checks/NDv4 into tests/health_checks/common and then make a parameterized call with sku name from tests/health_checks/NDv4

This will enable us to extend these health check scripts to other skus.
@jithinjosepkl @moes1 @jeseszhang1010 what do you guys think?

Reply:
My thought was that the different skus may need entirely different tests and so it might get confusing to put the tests all in one directory. GPU device to host bandwidth tests will not be applicable to skus without GPUs, for example. But I can see advantages of both ways of doing things.

Reply:
I'm okay with current folder org now. 
We do need some sort of refactoring to cover more SKUs though. which requires some design work to figure out what the common checks are for different SKUs, and how to organize the different checks for specific SKUs.